client/components/thank-you/index.tsx

```diff
@@ -34,12 +34,27 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
 	}
 `;
 
-const ThankYouHeader = styled.div`
+const ThankYouHeader = styled.div< ThankYouThemeProps >`
 	width: 100%;
-	height: 240px;
-	background-color: var( --studio-gray-0 );
 	display: flex;
 	justify-content: center;
+	background-color: ${ ( props ) => props.theme.colors.backgroundColorHeader };
+	height: 352px;
```

Comment:
I think this should be `min-height` - using `height` means that if we end up with really long text, the text starts to overflow the container.
```suggestion
	min-height: 352px;
```